Assets/Scripts/DayNightCycle/ToggleDayNightController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Assets.Scripts.Services;
+using Assets.Scripts.Settings;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+namespace Assets.Scripts.DayNightCycle
+{
+    public class ToggleDayNightController : MonoBehaviour
+    {
+        public Color DefaultSkyColor;
+        public Color DefaultEquatorColor;
+        public Color DefaultHorizonColor;
+        public GameObject DayNightControllerPrefab = null;
```

Comment:
DayNightControllerPrefab, PauseDayNightButton, DayNightController, MainLight, skyBoxReset, time and seconds are all unused.

Reply:
Yes will remove these.